ods_ci/tests/Resources/Page/ODH/JupyterHub/LoginJupyterHub.robot

```diff
@@ -13,15 +13,9 @@ Login To Jupyterhub
    ${login_required} =  Is OpenShift Login Visible
    IF  ${login_required}  Login To Openshift  ${ocp_user_name}  ${ocp_user_pw}  ${ocp_user_auth_type}
 
-Is Service Account Authorization Required
+Verify Service Account Authorization Not Required
    ${title} =  Get Title
-   ${result} =  Run Keyword And Return Status  Should Start With  ${title}  Authorize service account
-   RETURN  ${result}
-
-Authorize JupyterLab Service Account
-  Wait Until Page Contains  Authorize Access
-  Checkbox Should Be Selected  user:info
-  Click Element  approve
+   Should Not Start With  ${title}  Authorize service account
```

Comment:
Just want to assure that the returned value is the result of this keyword, correct? No need for the RETURN statement anymore indeed, right?

Reply:
Yeah, the (renamed) keyword now does its own assertion, no need to return anything.